.ci.yaml

```diff
@@ -3863,6 +3867,8 @@ targets:
   - name: Mac_arm64 module_test_ios
     recipe: devicelab/devicelab_drone
     timeout: 60
+    # Flake rate of >10% in presubmit
```

Comment:
for my own knowledge, how did you pull these numbers? Direct SQL queries to the buildbucket table or is there a dashboard?

Reply:
go/flutter-infra-metrics flakey test dashboard

Reply:
nice